docs/how_to/manage_tasks.md

```diff
@@ -0,0 +1,156 @@
+
+### Managing a task
+
+The `Task` class provides mechanisms to track and manage the status of a task.
+With a `Task` object at hand, the user can:
+ * Get the status, i.e., whether the task is running, has finished, has failed, etc;
+ (see section about [task lifecycle](#task-lifecycle) for more details);
+ * Get the machine type where it ran/is running;
+ * Get the execution time.
+ * Download either all or just a subset of the output files produced by the task;
+ * Turning it into a blocking call;
+ * Kill it;
+
+The following snippets show how to use the `Task` class to manage a task.
+
+```python
+# create a new task or retrieve a task from a previous session
+# task = simulator.run(...)
+#   or 
+# task = Task(i4ir3kvv62odsfrhko4y8w2an)
+
+# Get status of the task.
+>>> task.get_status()
+<DynamicSchema: 'success'>
+
+# Get the computational resource type where the task is
+# running or was submitted to
+>>> task.get_machine_type()
+'c2-standard-4'
+
+>>> task.get_computation_time()
+'00:03:12'
+
+# kill a task that hasn't completed yet
+>>> task.kill()
+
+# Get information about the output files generated by the simulation
+>>> output_files_info = task.get_output_files_info()
+>>> print(output_files_info)
+Archive size: 262.00 KB
+Contents:
+  Size         Compressed   Name
+  3.12 KB      859 B        stdout.txt
+  242 B        160 B        stderr.txt
+  5.76 KB      4.10 KB      vtk/ParticleData_Fluid_40.vtk
+  5.76 KB      4.09 KB      vtk/ParticleData_Fluid_32.vtk
+  5.76 KB      4.09 KB      vtk/ParticleData_Fluid_17.vtk
+  ...
+  4.98 KB      989 B        log/SPH_log.txt
+
+```
+
+Most of the methods of the `Task` class in the snippet above accept multiple
+arguments to configure the action to be performed. For example, the `download_outputs`
+method allows the user to specify which files to download and where to save
+the downloaded files.
+Please refer to the [Task API documentation](../api/inductiva.tasks.html) for more
```

Comment:
This should point to a readme about Task class in the user reference section, is that it?